.eslintrc.changed.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
     'jsx-a11y/anchor-is-valid': 2,
     'jsx-a11y/label-has-associated-control': 1,
     'jsx-a11y/no-static-element-interactions': 2,
+    '@department-of-veterans-affairs/prefer-telephone-component': 2,
+    '@department-of-veterans-affairs/telephone-contact-3-or-10-digits': 2,
```

Comment:
These will be switched back to errors in a week or so. I'm removing them now so that we can merge and make an announcement. The announcement and the warnings will give people a chance to fix the issues if they want before we start turning them into errors.